ProjectLighthouse.Servers.Website/Pages/UserPrivacyPage.cshtml

```diff
@@ -0,0 +1,107 @@
+﻿@page "/user/{userId:int}/privacy"
+@using LBPUnion.ProjectLighthouse.Extensions
+@using LBPUnion.ProjectLighthouse.Localization.StringLists
+@using LBPUnion.ProjectLighthouse.Types.Entities.Profile
+@using LBPUnion.ProjectLighthouse.Types.Users
+@model LBPUnion.ProjectLighthouse.Servers.Website.Pages.UserPrivacyPage
+
+@{
+    Layout = "Layouts/BaseLayout";
+    Model.Title = Model.Translate(ProfileStrings.Title, Model.ProfileUser!.Username);
+    Model.ShowTitleInPage = false;
+
+    bool isMobile = Request.IsMobile();
+}
+
+<div class="@(isMobile ? "" : "ui left aligned grid")">
+    <div class="column">
+        <h1>@Model.ProfileUser.Username's Privacy Settings</h1>
+        <form method="POST" class="ui form center aligned" action="/user/@Model.ProfileUser.UserId/privacy">
+            @Html.AntiForgeryToken()
+            <div class="ui yellow segment">
+                <h2><i class="ui icon comment alternate"></i> Profile Privacy</h2>
+                <div class="field">
+                    <label style="text-align: left" for="profilePrivacyLevel">Privacy Level</label>
+                    <select class="ui fluid dropdown" type="text" name="profilePrivacyLevel" id="profilePrivacyLevel">
+                        <option value="public" @(Model.ProfileUser.ProfileVisibility == PrivacyType.All ? "selected" : "")>
+                            Share your profile with everyone!
+                        </option>
+                        <option value="signedInOnly" @(Model.ProfileUser.ProfileVisibility == PrivacyType.PSN ? "selected" : "")>
+                            Only share your profile with users who are signed into the website or playing in-game.
+                        </option>
+                        <option value="inGameOnly" @(Model.ProfileUser.ProfileVisibility == PrivacyType.Game ? "selected" : "")>
+                            Only share your profile with users who are playing in-game.
+                        </option>
+                    </select>
+                </div>
+                <div class="field">
+                    <label style="text-align: left" for="profileCommentsEnabled">
+                        Comments
+                        @if (Model.CommentsDisabledByModerator)
+                        {
+                            <small style="color: red">Locked by a moderator</small>
```

Comment:
Does Fomantic have classes for formatting text so we don't have to use inline styles?